### PR TITLE
luci-app: Improve zh-CN translations

### PR DESCRIPTION
**Pull request by liushuyu**
This change is [Reviewable](https://reviewable.io/reviews/openwrt/luci/1111)

### CODE REVIEW

File: applications/luci-app-adblock/po/zh-cn/adblock.po [hunk 1]
```diff
@@ -80,18 +81,18 @@ msgid "Enable blocklist backup"
...
 msgid ""
-"For SSL protected blocklist sources you need a suitable SSL library, e.g. "
-"'libustream-ssl' or the wget 'built-in'."
+"For SSL protected blocklist sources you need a suitable SSL library, e.g."
+" 'libustream-ssl' or the wget 'built-in'."
 msgstr ""
 
 msgid "For further information"
-msgstr ""
+msgstr "更多详细信息"
```
Comment (yousong) [inline applications/luci-app-adblock/po/zh-cn/adblock.po]: “更多信息”即可

File: applications/luci-app-clamav/po/zh-cn/clamav.po [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+msgstr "跟随目录符号链接"
+
+msgid "Follow file symlinks"
+msgstr "跟随文件符号链接"
+
+msgid "Log"
+msgstr "日志"
+
+msgid "Log additional infection info"
+msgstr "记录文件感染的详细信息"
```
Comment (yousong) [inline applications/luci-app-clamav/po/zh-cn/clamav.po]: "文件"可省略

File: applications/luci-app-clamav/po/zh-cn/clamav.po [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+msgid "Log time with each message"
+msgstr "记录每条信息的时间"
+
+msgid "Max directory scan depth"
+msgstr "最大目录扫描深度"
```
Comment (yousong) [inline applications/luci-app-clamav/po/zh-cn/clamav.po]: “目录”可省略

File: applications/luci-app-clamav/po/zh-cn/clamav.po [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+msgstr "跟随文件符号链接"
+
+msgid "Log"
+msgstr "日志"
+
+msgid "Log additional infection info"
+msgstr "记录文件感染的详细信息"
+
+msgid "Log time with each message"
+msgstr "记录每条信息的时间"
```
Comment (yousong) [inline applications/luci-app-clamav/po/zh-cn/clamav.po]: 可改为“记录消息时间戳”

File: applications/luci-app-statistics/po/zh-cn/rrdtool.po [hunk 5]
```diff
@@ -61,299 +62,299 @@ msgstr "响应"
...
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:11
 msgid "stat_dg_title_iptables__ipt_packets"
-msgstr "iptables包"
+msgstr "%H: 防火墙 - 已处理的数据包"
 
 #. Packets/s
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:12
 msgid "stat_dg_label_iptables__ipt_packets"
-msgstr ""
+msgstr "数据包/秒"
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/rrdtool.po]: 可不翻译

File: applications/luci-app-statistics/po/zh-cn/rrdtool.po [hunk 6]
```diff
@@ -61,299 +62,299 @@ msgstr "响应"
...
 
 #. Chain \"%di\"
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:13
 msgid "stat_ds_ipt_packets"
-msgstr ""
+msgstr "Chain \\\"%di\\\""
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/rrdtool.po]: 可维持原样？

File: applications/luci-app-statistics/po/zh-cn/rrdtool.po [hunk 7]
```diff
@@ -61,299 +62,299 @@ msgstr "响应"
...
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:15
 msgid "stat_dg_label_netlink__if_octets"
-msgstr ""
+msgstr "字节/秒"
 
 #. Bytes (%ds)
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:16
 msgid "stat_ds_if_octets"
-msgstr ""
+msgstr "字节 (%ds)"
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/rrdtool.po]: 同上

File: applications/luci-app-statistics/po/zh-cn/rrdtool.po [hunk 8]
```diff
@@ -61,299 +62,299 @@ msgstr "响应"
...
 
 #. %H: Netlink - Packets on %pi
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:17
 msgid "stat_dg_title_netlink__if_packets"
-msgstr ""
+msgstr "%H: 网络 - %pi 的数据包"
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/rrdtool.po]: If netlink here refers to the Linux netlink specifically, we should keep it as is.

File: applications/luci-app-statistics/po/zh-cn/rrdtool.po [hunk 9]
```diff
@@ -61,299 +62,299 @@ msgstr "响应"
...
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:19
 msgid "stat_ds_if_packets"
-msgstr ""
+msgstr "已处理 (%ds)"
 
 #. Dropped   (%ds)
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:20
 msgid "stat_ds_if_dropped"
-msgstr ""
+msgstr "丢失 (%ds)"
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/rrdtool.po]: 可改为“丢弃”

File: applications/luci-app-statistics/po/zh-cn/rrdtool.po [hunk 10]
```diff
@@ -61,299 +62,299 @@ msgstr "响应"
...
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:26
 msgid "stat_dg_label_netlink__if_collisions"
-msgstr ""
+msgstr "碰撞/秒"
 
 #. Collisions
 #: applications/luci-statistics/luasrc/i18n/rrdtool.en.lua:27
 msgid "stat_ds_if_collisions"
-msgstr ""
+msgstr "碰撞"
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/rrdtool.po]: To be honest, I can search the net to see what "collisions" means, but it's very likely that people may have no clue what the translated word refers to.

People looking at these statistics are very likely already familiar with outputs from tools like ifconfig, ip link, sar, etc.  So I think we should just keep them as is.

File: applications/luci-app-statistics/po/zh-cn/statistics.po [hunk 11]
```diff
@@ -503,7 +505,7 @@ msgstr "OLSRd插件通过txtinfo获取meshed网络信息。"
 msgid ""
 "The OpenVPN plugin gathers information about the current vpn connection "
 "status."
-msgstr ""
+msgstr "OpenVPN 插件可以获取当前 VPN 连接状态。"
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/statistics.po]: 可改为“OpenVPN 插件可以获取 VPN 连接当前状态。”

File: applications/luci-app-statistics/po/zh-cn/statistics.po [hunk 12]
```diff
@@ -629,6 +631,9 @@ msgid ""
 "to gather data and <a href=\"http://oss.oetiker.ch/rrdtool/\">RRDtool</a> to "
 "render diagram images."
 msgstr ""
+"统计数据软件包使用 <a href=\"https://collectd.org/\">Collectd</a> 来收集数"
+"据，并用 <a href=\"http://oss.oetiker.ch/rrdtool/\">RRDtool</a> 渲染统计图"
+"表。"
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/statistics.po]: “统计数据软件包”可改为“软件包statistics”，“渲染统计图表”可改为“生成统计图表”

File: applications/luci-app-statistics/po/zh-cn/statistics.po [hunk 13]
```diff
@@ -640,6 +645,8 @@ msgid ""
 "read from /sys/class/thermal/*/temp ( '*' denotes the thermal device to be "
 "read, e.g. thermal_zone1 )"
 msgstr ""
+"温感插件将会监控系统温度。数据主要是通过 /sys/class/thermal/*/temp ('*' 表示"
+"温感设备的名字，比如 thermal_zone1) 读取的。"
```
Comment (yousong) [inline applications/luci-app-statistics/po/zh-cn/statistics.po]: 应尽量避免句中的括号，不利于流畅阅读。可改为，“数据取自 /sys/.../temp （explain *）”

File: applications/luci-app-wireguard/po/zh-cn/wireguard.po [hunk 14]
```diff
@@ -0,0 +1,73 @@
...
+msgstr "传输端点"
+
+msgid "Firewall Mark"
+msgstr "防火墙标识"
+
+msgid "Interface"
+msgstr "接口"
+
+msgid "Interface does not have a public key!"
+msgstr "界面没有公钥！"
```
Comment (yousong) [inline applications/luci-app-wireguard/po/zh-cn/wireguard.po]: Interface这里是否指“网卡”？

File: applications/luci-app-wireguard/po/zh-cn/wireguard.po [hunk 15]
```diff
@@ -0,0 +1,73 @@
...
+
+msgid "Latest Handshake"
+msgstr "最新的握手"
```
Comment (yousong) [inline applications/luci-app-wireguard/po/zh-cn/wireguard.po]: Lastest这里可以译为“上次”

File: applications/luci-app-wireguard/po/zh-cn/wireguard.po [hunk 16]
```diff
@@ -0,0 +1,73 @@
...
+
+msgid "Listen Port"
+msgstr "监听端口"
+
+msgid "Never"
+msgstr "从不"
+
+msgid "Peer"
+msgstr "同级"
```
Comment (yousong) [inline applications/luci-app-wireguard/po/zh-cn/wireguard.po]: 译为“对端”？

File: applications/luci-app-wireguard/po/zh-cn/wireguard.po [hunk 17]
```diff
@@ -0,0 +1,73 @@
...
+
+msgid "Persistent Keepalive"
+msgstr "持久性 Keepalive"
```
Comment (yousong) [inline applications/luci-app-wireguard/po/zh-cn/wireguard.po]: 可译为“Keepalive间隔（秒）“

**Comment by hnyman** [comment]
@yousong  
any comments to changes to the PR?

File: applications/luci-app-wireguard/po/zh-cn/wireguard.po (hunk 14 above)
Comment (yousong) [inline applications/luci-app-wireguard/po/zh-cn/wireguard.po]: @liushuyu , 此处Interface译为“界面”不合日常使用习惯，但是”网卡“直接理解一般指物理网卡；还有一些非直译的方式，比如，”接口未配置公钥“，“接口公钥配置缺失”，其中，“接口”也可以替换为“wireguard接口“、”wg接口“、“网络接口”

**Comment by yousong** [comment]
@hnyman there is just a single issue remained, probably missed out in the last round of updates.

@liushuyu please rebase and quash these commits into a single one after the review is done ;)

**Comment by liushuyu** [comment]
>@liushuyu please rebase and quash these commits into a single one after the review is done ;)

Rebased and squashed

**Comment by yousong** [comment]
LGTM, thanks

**Pull request merged**